EKF/control.cpp

```diff
@@ -43,6 +43,8 @@
 
 void Ekf::controlFusionModes()
 {
+	// Store the status to enable change detection
```

Comment:
@priseborough I believe there was a bug in the old code where the old status flags were set to the current status flags at the end of the control file which does not make any sense in my opinion. Do you agree or what what the intention to do this?